docs/solana/migration/migration-status.mdx

```diff
@@ -0,0 +1,47 @@
+---
+id: migration-status
+title: Helium Migration Status Service
+pagination_label: Migration Status
+sidebar_label: Migration Status
+description: Helium Migration Status Service Documentation
+image: https://i.imgur.com/bWAbESE.png
+slug: /solana/migration/migration-status
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl'
+
+The Solana Migration Status service can be leveraged in order to appropriately feature flag 
+attributes in an application. For example, this service can be used to disable transactions during 
+the migration or to point users to separate services as the migration completes.
+
+## Migration States
+
+The three possible values that may be returned by the API represent different stages of the 
+migration process:
+
+- **`not_started`**: This means that the migration process has not yet begun. The Helium blockchain 
+is still operating and the migration to Solana has not yet been initiated.
+- **`in_progress`**: This means that the migration process is currently underway. The Helium 
+blockchain is in the process of being migrated to Solana. The Helium Blockchain will be halted 
+during this process. Proof of Coverage and Data Transfer activity continue to function throughout 
+the migration process.
+- **`complete`**: This means that the migration process has been completed successfully. The Helium 
+blockchain has been migrated to Solana. The `finalBlockHeight` value will update from 
+`null` to the final block height of the Helium Blockchain.
+
+
+## Example Usage
+
+```bash
+curl https://solana-status.helium.com
```

Comment:
since this is an API, we should probably move this to .io

Reply:
should we hold off on publishing for this?

Reply:
yes, hold off for now. Would like to get the .io sorted.
Additionally, it might make more sense for this page to be a `readme` on the repo with only light references from docs.